app/javascript/angular/code/calculateBounds.js

```diff
@@ -0,0 +1,38 @@
+import { pixelsToLength } from "./mapsUtils.js";
+
+// The graph is ~220px high. For whatever reason the OFFSET_IN_PIXEL
+// must be 1000 to not have the session path covered by the graph.
+const OFFSET_IN_PIXEL = 1000;
```

Comment:
🤔 will take a look when/if I style anything there

Reply:
No worries it's gone ;)